apps/sms/views/conversation/js/information.js

```diff
@@ -247,14 +248,17 @@ var VIEWS = {
 
         ConversationView.promptContact({
           number: target.dataset.number
-        });
+        }).then(
+          () => Navigation.toPanel('thread', { id: this.id })
+        );
```

Comment:
uploaded to https://wiki.mozilla.org/images/b/bb/HTML5_SMS-MMSUserStorySpecifications_20130503_V8.0.pdf